src/promptflow-evals/promptflow/evals/evaluate/_eval_run.py

```diff
@@ -249,6 +249,8 @@ def get_metrics_url(self):
     def _get_token(self):
         # We have to use lazy import because promptflow.azure
         # is an optional dependency.
+        from promptflow.azure._utils._token_cache import ArmTokenCache
+        return ArmTokenCache().get_token(self._ml_client._credential)
         return self._ml_client._credential.get_token(EvalRun._SCOPE)

```

Comment:
We can remove this line.

Reply:
It used to return token object, now it returns token string, so we also need to remove ".token" from where _get_token() is called.